arcade/sprite_list/billboard_list.py

```diff
@@ -0,0 +1,39 @@
+from arcade.sprite_list import SpriteList
+
+
+class BillboardList(SpriteList):
+    """
+    The purpose of the BillboardList is to batch draw a list of sprites.
+    The difference between the BillboardList and a basic SpriteList is how it
+    orientates Sprites. The normal SpriteList always aligns them to the X-Y plane.
+    The BillboardList purposely forces the sprite to always look directly towards
+    the screen.
```

Comment:
I'd clean up the intro section as follows.
```suggestion
    """Draw sprites to always face the screen regardless of camera orientation.

    Since the basic :py:class:`~arcade.SpriteList` draws every :py:class:`~arcade.Sprite`
    as aligned with the X and Y axes, it allows perspective and camera position to affect
    how sprite polygons are drawn. This class differs by using a special default shader
    program to always make sure the sprite polygons face the camera. This is also known
    as billboarding or billboard-style sprites.
```